xbmc/platform/win32/PlatformWin32.cpp

```diff
@@ -56,3 +56,12 @@ void CPlatformWin32::PlatformSyslog()
 {
   CWIN32Util::PlatformSyslog();
 }
+
+bool CPlatformWin32::SupportsUserInstalledBinaryAddons()
+{
+#if defined(TARGET_WINDOWS_STORE)
```

Comment:
`PlatformWin32.cpp` is only for Windows desktop. Is not need discriminate `TARGET_WINDOWS_STORE` here.

Windows Store specific overrides are on:
https://github.com/xbmc/xbmc/blob/master/xbmc/platform/win10/CPUInfoWin10.h
https://github.com/xbmc/xbmc/blob/master/xbmc/platform/win10/PlatformWin10.cpp

Reply:
You are the boss. 

I thought, because there are NO occurances of TARGET_WINDOWS_STORE in platform/win10, but many in platform/win32, the latter would be the right place.

```log
./platform/win32/PlatformWin32.cpp:#if defined(TARGET_WINDOWS_STORE)
./platform/win32/pch.h:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/WIN32Util.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/filesystem/Win32File.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/filesystem/Win32File.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/filesystem/Win32File.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/filesystem/Win32File.cpp:#ifdef TARGET_WINDOWS_STORE
./platform/win32/filesystem/Win32File.cpp:#ifdef TARGET_WINDOWS_STORE
```

Reply:
fixed